quinn/src/connection.rs

```diff
@@ -897,6 +908,7 @@ impl ConnectionRef {
                 runtime,
                 send_buffer: Vec::new(),
                 buffered_transmit: None,
+                observed_external_addr: watch::Sender::new(None),
```

Comment:
```suggestion
                observed_external_addr: watch::channel(None).0,
```

Reply:
I'm very confused this compiles at all in CI without this change?  Locally this straight out didn't compile for me with `Sender::new`.  Am I holding this wrong?

Reply:
yeah I was going to ask you about this.. why create a receiver to drop it immediately? compiles fine for me